### PR TITLE
PCH: Fix some untranslatable strings

### PR DESCRIPTION
**Pull request by acicovic**
## Description
This PR fixes a couple of strings that couldn't be translated, so they are translatable now.

## Motivation and context
All our strings should be translatable.

## How has this been tested?
Untested.

### CODE REVIEW

**Comment by acicovic** [comment]
@vaurdan, on second pass, these strings seem a bit incorrect and would need to be updated?

**Comment by acicovic** [comment]
How about updating the string to "Settings"?

**Comment by vaurdan** [comment]
Yeah, they are definitely incorrect. Those are some left-over strings from when I was experimenting with the component before the final implementation. Changing them to `Settings` is definitely more appropriate :)

File: src/content-helper/editor-sidebar/performance-details/component-panel.tsx [hunk 1]
```diff
@@ -63,7 +64,7 @@ export const PerformanceStatPanel = (
 				{ ( controls && ! dropdownChildren ) && (
 					<DropdownMenu
 						icon={ icon }
-						label="Select a direction"
+						label={ __( 'Select a direction', 'wp-parsely' ) }
```
Comment (vaurdan) [inline src/content-helper/editor-sidebar/performance-details/component-panel.tsx]:
```suggestion
						label={ __( 'Settings', 'wp-parsely' ) }
```

File: src/content-helper/editor-sidebar/performance-details/component-panel.tsx [hunk 2]
```diff
@@ -73,7 +74,7 @@ export const PerformanceStatPanel = (
 				{ dropdownChildren && (
 					<DropdownMenu
 						icon={ icon }
-						label="Select a direction"
+						label={ __( 'Select a direction', 'wp-parsely' ) }
```
Comment (vaurdan) [inline src/content-helper/editor-sidebar/performance-details/component-panel.tsx]:
```suggestion
						label={ __( 'Settings', 'wp-parsely' ) }
```

**Review by vaurdan — APPROVED** [review]

**Pull request merged**